dss-core/src/test/java/io/github/ztkmkoo/dss/core/actor/blocking/jdbc/DssDatabaseServiceActorTest.java

```diff
@@ -0,0 +1,64 @@
+package io.github.ztkmkoo.dss.core.actor.blocking.jdbc;
+
+import java.io.IOException;
+
+import org.junit.jupiter.api.Test;
+
+import com.healthmarketscience.sqlbuilder.BinaryCondition;
+import com.healthmarketscience.sqlbuilder.DeleteQuery;
+import com.healthmarketscience.sqlbuilder.InsertQuery;
+import com.healthmarketscience.sqlbuilder.SelectQuery;
+import com.healthmarketscience.sqlbuilder.UpdateQuery;
+
+import io.github.ztkmkoo.dss.core.actor.AbstractDssActorTest;
+
+public class DssDatabaseServiceActorTest extends AbstractDssActorTest {
+	
+	private static final String testTable = "test_table";
+
+    @Test
+    void selectTest() throws IOException {
+        SelectQuery query = new SelectQuery()
+                .addCustomFromTable(testTable)
+                .addCustomColumns("name", "age")
+                .addCondition(BinaryCondition.equalTo("name", "JuHyun"))
+                .addCondition(BinaryCondition.equalTo("age", "10"))
+                .validate();
+        
+        String statement = query.toString();
+        System.out.println(statement);
```

Comment:
Hi, @JuHyun419 
Why not assert this statement?
Like assertEquals("SELECT name, age FROM test_table WHERE name = 'JuHyun' AND age = '10'", statement);
And also other test.

Reply:
Okay~ I'll update 👍